build.sbt

```
@@ -2,8 +2,8 @@ import Dependencies._
 
 ThisBuild / scalaVersion     := "2.13.8"
 ThisBuild / version          := "0.1.0-SNAPSHOT"
-ThisBuild / organization     := "com.example"
-ThisBuild / organizationName := "example"
+ThisBuild / organization     := "com.buildo"
```

```suggestion
ThisBuild / organization     := "io.buildo"
```
(pignoleria a caso, sorry, ovviamente non è davvero rilevante, però in generale ha senso usare domini veri)

---

Ottimo catch, modifica integrata =)